python/paddle/fluid/tests/unittests/test_multi_tensor_adam.py

```diff
@@ -0,0 +1,387 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import paddle
+import paddle.nn as nn
+import numpy as np
+
+
+class MLPLayer(nn.Layer):
+
+    def __init__(self, input_size, hidden_size, output_size, n):
+        super(MLPLayer, self).__init__()
+        self.linear_first = nn.Linear(input_size, hidden_size)
+        self.linear_mid = nn.Sequential(
+            ('l0', nn.Linear(hidden_size, hidden_size)))
+        for i in range(n - 1):
+            self.linear_mid.add_sublayer('l' + str(i + 1),
+                                         nn.Linear(hidden_size, hidden_size))
```

Comment:
Why not just `self.linear_mid = nn.Sequential([nn.Linear(hidden_size, hidden_size) for _ in range(n)])`